.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
```

Comment:
you have to delete the .idea folder in your project before pushing the code to the repo. Ensure the files that are reflected in the git repo does not reflect any idea file

Reply:
deleted